src/main/webapp/app/pages/curation/collapsible/CancerTypeCollapsible.tsx

```diff
@@ -49,14 +49,17 @@ function CancerTypeCollapsible({
   deleteSection,
   isGermline,
 }: ICancerTypeCollapsibleProps) {
-  const [cancerTypes, setCancerTypes] = useState<CancerType[]>(null);
-  const [cancerTypesUuid, setCancerTypesUuid] = useState<string>(null);
-  const [cancerTypesReview, setCancerTypesReview] = useState<Review>(null);
+  const [cancerTypes, setCancerTypes] = useState<CancerType[] | null>(null);
+  const [cancerTypesUuid, setCancerTypesUuid] = useState<string | null>(null);
+  const [cancerTypesReview, setCancerTypesReview] = useState<Review | null>(null);
```

Comment:
null here too. Might be to use undefined by default?

Reply:
Yeah I think it's more proper react to have the implied `undefined` instead of passing a `null`.